src/middlewared/middlewared/client/client.py

```diff
@@ -47,6 +49,20 @@ def wait(self, timeout=None):
 
 CALL_TIMEOUT = int(os.environ.get('CALL_TIMEOUT', 60))
 
+class ReservFDError(Exception):
```

Comment:
Reserv**e**FDError?

Reply:
Kept the name inline with the message that was raised (i.e `Failed to reserv privileged port`). I'll correct the spelling.